src/main/java/net/javadiscord/javabot/systems/commands/ChangeMyMindCommand.java

```diff
@@ -24,14 +24,26 @@
  */
 @Deprecated
 public class ChangeMyMindCommand implements SlashCommand {
+	/**
+	 * The maximum acceptable length for texts to send to the API. Technically,
+	 * the API supports up to but not including 2000, but we'll make that much
+	 * lower to avoid issues, since the API fails with a 501 error. People
+	 * shouldn't really be putting paragraphs into this anyway.
+	 */
+	private static final int MAX_SEARCH_TERM_LENGTH = 1500;
+
 	@Override
 	public ReplyCallbackAction handleSlashCommandInteraction(SlashCommandInteractionEvent event) {
 		var hook = event.getHook();
-		String encodedSearchTerm = null;
+		String encodedSearchTerm;
 		try {
 			encodedSearchTerm = URLEncoder.encode(Objects.requireNonNull(event.getOption("text")).getAsString(), StandardCharsets.UTF_8.toString());
 		} catch (UnsupportedEncodingException e) {
 			e.printStackTrace();
+			return event.reply("The text you provided is using an unsupported encoding.");
```

Comment:
What is that supposed to be? The error occurs when `UTF-8` is unknown.

Reply:
Removed it as `URLEncoder#encode` doesn't even throw that when given a Charset instead of a String.